app/views/candidate_mailer/_new_flow_chase_reference.text.erb

```diff
@@ -11,4 +11,4 @@ You can sign into your account to:
 
 [Sign into your account](<%= candidate_magic_link(@application_form.candidate) %>).
 
-<%= @application_form.application_choices.pending_conditions.first.provider.name %> must check your references before they can confirm your place on the course. Contact them if you need help getting references or choosing who to ask.
\ No newline at end of file
+<%= @application_form.application_choices.pending_conditions.first&.provider&.name %> must check your references before they can confirm your place on the course. Contact them if you need help getting references or choosing who to ask.
```

Comment:
Wouldn't this be blank and read `[blank]  must check your references` if any of these `&`s failed?

Reply:
it would, yes. but i don't think this can ever happen outside the context of a test. checking now.

Reply:
fyi @duncanjbrown i reverted this change and timecopped the corresponding test